src/ApacheThriftLinter.php

```diff
@@ -104,6 +104,10 @@ public function getInstallInstructions() {
     if ($this->installInstructions) {
       return $this->installInstructions;
     }
+    $instructions = parent::getInstallInstructions();
+    if (!empty($instructions)) {
+      return $instructions;
+    }
```

Comment:
This could also become:

```php
$instructions = nonempty($this->installInstructions, parent::getInstallInstructions());
if (!empty($instructions)) {
  return $instructions;
}
```

... but we can probably also remove the linter-local `installInstructions` because it was only recently introduced, too.